src/vm/arm/asmhelpers.S

```diff
@@ -741,6 +741,49 @@ NESTED_ENTRY ProfileLeaveNaked, _TEXT, NoHandler
     EPILOG_POP "{r1, r2, r4, r5, r7, r11, pc}"
 NESTED_END ProfileLeaveNaked, _TEXT
 
+//
+// EXTERN_C void ProfileTailcallNaked(FunctionIDOrClientID functionIDOrClientID);
+//
+NESTED_ENTRY ProfileTailcallNaked, _TEXT, NoHandler
+    PROLOG_PUSH "{r1, r2, r4, r5, r7, r11, lr}"
+    PROLOG_STACK_SAVE_OFFSET r7, #16
+
+    // fields of PLATFORM_SPECIFIC_DATA, in reverse order
```

Comment:
The struct name is  `PROFILE_PLATFORM_SPECIFIC_DATA`, not `PLATFORM_SPECIFIC_DATA`. And the comment "in reverse order" is misleading. The fields are in the order that you've listed them below. I guess you've meant that they are pushed in the reverse order by that comment, right?

Reply:
Fixed